src/types.ts

```diff
@@ -1679,7 +2024,29 @@ export interface KAPLAYCtx<
     ): KEventController;
     /**
      * Register an event that runs every frame when any gamepad buttons are held down.
-     *
+     * 
+     * @example
+     * ```js
+     * // basic car with brakes
+     * let car = add([
```

Comment:
Same about comps, I think we should mantain simple the ones in controller, I like these examples but can be  real examples, not the example of the function, should be simpler